frontend/src/styles/theme.ts

```diff
@@ -0,0 +1,19 @@
+import { extendTheme } from "@chakra-ui/react";
+
+import colors from "./colors";
+// import textStyles from "./textStyles";
+
+const customTheme = extendTheme({
+  fonts: {
+    // figure out how to get system fonts here
+    // heading: "Inter, sans-serif",
+    // body: "Inter, sans-serif",
+  },
+  // textStyles,
+  colors,
+  config: {
+    initialColorMode: "light",
+  },
+});
+
+export default customTheme;
```

Comment:
I think these files are relics from when you implemented the chakra theme - could we delete this folder?